application/common/mail/refresh-backup-codes.text.php

```diff
@@ -21,8 +21,25 @@
  * @var string $supportEmail
  * @var string $supportName
  * @var bool   $isMfaEnabled
+ * @var int    $numRemainingCodes
  */
 ?>
 Dear <?= $displayName ?>,
 
-TODO: Compose contents of this email message.
\ No newline at end of file
+You only have <?= $numRemainingCodes ?> Printable Codes remaining for use with 2-Step Verification on your
```

Comment:
As above, possibly change `Printable Codes` to `Printable Code(s)` (depending on what you decide for the HTML format of this email).